### PR TITLE
File name changes for organization, and staging release pipeline

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

*Issue #, if available:*

*Description of changes:*

Long Overdue name/organization changes to make more sense what each file is doing, and adding a set of staging files for the new staging pipeline we are building.

### CODE REVIEW

**Comment by jonahjon** [comment]
We'll want to Merge this and CR-78338072 close together ideally so will wait until both get approved to un-hold and merge.

File: generatebundlefile/hack/release_staging.sh [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    awsAuth "$REPO" | "$ORAS_BIN" login "$REPO" --username AWS --password-stdin
+    "$ORAS_BIN" push "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
+    "$ORAS_BIN" push "${REPO}:v${version}-latest" bundle.yaml
+}
+
+for version in 1-20 1-21 1-22 1-23 1-24; do
+    generate ${version}
+done
+
+export AWS_PROFILE=prod
```
Comment (ewollesen) [inline generatebundlefile/hack/release_staging.sh:95]: Is prod the correct profile here? If so, a comment might be warranted.

Comment (jonahjon) [reply generatebundlefile/hack/release_staging.sh]: Fixed

**Pull request merged**